README.md

```diff
@@ -57,9 +57,9 @@ Note that the external-provisioner does not scale with more replicas. Only one e
 
 * `--worker-threads <num>`: Number of simultaneously running `ControllerCreateVolume` and `ControllerDeleteVolume` operations. Default value is `100`.
 
-* `--kube-api-qps <num>`: QPS for clients that communicate with the kubernetes apiserver. Defaults to `5.0`.
+* `--kube-api-qps <num>`: [QPS](https://github.com/kubernetes/client-go/blob/release-1.19/rest/config.go#L115) for clients that communicate with the Kubernetes apiserver. Defaults to `5.0`.
```

Comment:
`release-1.19` is still a branch. Better use a tag like `v0.19.0-rc.4`. Highlighting also becomes nicer when using a range: https://github.com/kubernetes/client-go/blob/5521967004d84d9e6f89df86dfeb5977f993bcbd/rest/config.go#L115-L117

But that's mute because that documentation still doesn't explain what QPS and burst are. Explaining "QPS" with "QPS indicates the maximum QPS" doesn't help when it's unknown what "QPS" is in the first place :cry: 

Can you ask around whether there's any better documentation of these concepts?

Reply:
If we can't find better documentation, can you just add a high level description?

Reply:
How about this:

```
QPS - "The number of requests per second processed by a client to the Kubernetes API server"
Burst - "The number of requests to the Kubernetes API server, exceeding the QPS, that can be processed at any given time"
```

and i'll add a link to https://github.com/kubernetes/client-go/blob/master/util/flowcontrol/throttle.go#L48

Reply:
Sounds good to me. Just a nit, instead of "a client", what about "the Kubernetes client"

Reply:
I've been looking into this a bit more recently myself. I think a more accurate verb is "sent" instead of "processed", because that's really where the rate check happens (https://github.com/kubernetes/client-go/blob/5521967004d84d9e6f89df86dfeb5977f993bcbd/rest/request.go#L847-L852): before sending the request to the server, the client voluntarily slows itself down.

Reply:
@msau42 @pohly PTAL!